packages/dialog/src/index.tsx

```diff
@@ -253,6 +268,10 @@ export type DialogProps = {
    * @see Docs https://reacttraining.com/reach-ui/dialog#dialogoverlay-initialfocusref
    */
   initialFocusRef?: React.RefObject<any>;
+  /**
+   * This ref can help you to override the returnFocus of the FocusLock component
+   */
```

Comment:
What do you think about making this doc comment more similar to the `initialFocusRef` comment, since they are essentially inverses of each other?

"By default the element that opened the dialog will receive focus when the dialog closes, but you can provide a ref to focus instead."